docs/source/examples/assignment_workflows/plot_route_choice.py

```diff
@@ -51,7 +64,8 @@
 project.network.graphs.keys()
 
 # let's say we want to minimize the distance
-graph.set_graph("distance")
+graph.network = graph.network.assign(utility=graph.network.distance * theta)
```

Comment:
@janzill , this is what we were missing.

@Jake-Moss , we could have a method in the graph object to allow for a dictionary to be provided while we do the math in the background, but I don't think that's needed

Reply:
> @Jake-Moss , we could have a method in the graph object to allow for a dictionary to be provided while we do the math in the background, but I don't think that's needed

Not 100% sure what you mean here sorry? A dictionary to do what? Describe some function between columns?